lib/rubocop/cop/style/if_unless_modifier.rb

```diff
@@ -26,6 +26,7 @@ def on_if(node)
           return if if_else?(node)
           return if node.chained?
           return unless fit_within_line_as_modifier_form?(node)
+          return if node.descendants.any? { |d| d.type == :if }
```

Comment:
I think in practice there will be a single descendant since we already discard bodies with several statements ([per this existing test](https://github.com/amuino/rubocop/blob/if-unless-modifier-with-nested-conditional/spec/rubocop/cop/style/if_unless_modifier_spec.rb#L43-L54)) , so I'll change that.